app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] As it is [requested](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md) for this project, " `Modify the views in devise for registration and login to match the wireframes from the Sneak Peek and your styling.` ", please ensure the views in devise for the registration and login have been modified to match the [wireframes](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/sneak_peek.md)(`checkout sneak peek: Blog app`) and their styling.